plugins/toolbox/src/components/Root/tools.tsx

```diff
@@ -304,6 +306,20 @@ export const defaultTools: Tool[] = [
     category: 'Validate',
     description: 'Validates IBAN based on ISO 13616',
   },
+  {
+    id: 'pzn',
+    name: 'PZN validator',
+    component: <PznValidator />,
+    category: 'Validate',
+    description: 'Validates PZN (8-digits) from german pharmaceutical system',
```

Comment:
Sorry but I think this is not very widely used developer tool. In case you need this in your backstage instance, you can create the tool in your own repo and pass it to `additionalTools`

Reply:
pzn-validator is dropped.